db/structure.sql

```diff
@@ -177,7 +177,6 @@ CREATE TABLE public.activity_logs (
 --
 
 CREATE SEQUENCE public.activity_logs_id_seq
-    AS integer
```

Comment:
I'm just curious why there are so many changes in this file.

Reply:
This gets auto generated whenever you run a migration and the content is based on the shape of the database it is run against.  Depending on who ran it last, it can change somewhat dramatically.